gui/cppcheck_de.ts

```diff
@@ -122,63 +122,63 @@ Parameter: -l(line) (file)
     <message>
         <location filename="compliancereportdialog.ui" line="14"/>
         <source>Compliance Report</source>
-        <translation type="unfinished"></translation>
+        <translation>Compliance-Report</translation>
     </message>
     <message>
         <location filename="compliancereportdialog.ui" line="29"/>
         <source>Project name</source>
-        <translation type="unfinished"></translation>
+        <translation>Projektname</translation>
     </message>
     <message>
         <location filename="compliancereportdialog.ui" line="22"/>
         <source>Project version</source>
-        <translation type="unfinished"></translation>
+        <translation>Projekt-Version</translation>
     </message>
     <message>
         <location filename="compliancereportdialog.ui" line="42"/>
         <source>Coding Standard</source>
-        <translation type="unfinished"></translation>
+        <translation>Programmierstandards</translation>
     </message>
     <message>
         <location filename="compliancereportdialog.ui" line="50"/>
         <source>Misra C</source>
-        <translation type="unfinished"></translation>
+        <translation>Misra C</translation>
```

Comment:
For future improvements. We should have this (and the ones below) as non-`tr` strings as these are obviously do not need to be translated.

I filed https://trac.cppcheck.net/ticket/14345 about it.